### PR TITLE
Cleaned up files in sdk/io 

### PR DESCRIPTION
**Pull request by marcusnaslund**
Before you throw this out the window and yell at me, append `?w=1` :)

It's not completely magic compliant yet, but it's better to get all of these whitespace fixes out of the way first.

No functionality has changed. Part of the work on #692

### CODE REVIEW

File: source/sdk/io/File.ooc [hunk 1]
```diff
@@ -18,604 +18,603 @@ import text/StringTokenizer
...
+
+	extension: String { get { getExtension() } }
+	nameWithoutExtension: String { get { getNameWithoutExtension() } }
+	pathWithoutExtension: String { get { getPathWithoutExtension() } }
+	rectifySeparator: func {
+		if (this dir?() && !this path endsWith?(This separator)) {
 			oldPath := this path
-			this path = oldPath + File separator
+			this path = oldPath + This separator
 			oldPath free()
```
Comment (davidhesselbom) [inline source/sdk/io/File.ooc]: `this path = this path >> This separator` should work as a replacement to the 3 lines here.

Comment (davidhesselbom) [inline source/sdk/io/File.ooc]: But, it can be fixed later.

**Comment by davidhesselbom** [comment]
> append ?w=1

Nifty, innit? :)

File: source/sdk/io/File.ooc (hunk 1 above)
Comment (marcusnaslund) [inline source/sdk/io/File.ooc]: I'll write it up.

**Pull request merged**